website/content/docs/enterprise/license/faq.mdx

```diff
@@ -20,6 +20,59 @@ This check is part of the server boot-up process.
 In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol.
 This will no longer work since each server must be able to find a valid license during the startup process.
 
+## Q: What are the upgrade requirements?
+
+If ACLs are enabled, customers on Consul Enterprise 1.8+ or 1.9+ must upgrade Consul agents to the latest patch available (1.8.13 or 1.9.7) prior to upgrading the Consul server instances to 1.10.0.
+Otherwise, the Consul agents will fail to retrieve the license with a valid agent token.
+
+The following table provides an overview of 1.10.0 upgrade scenarios.
+
+| Client Version | Server Version | ACLs     | Snapshot Agent Version                             | Guidance                                                                                                                                      |
+| -------------- | -------------- | -------- | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1.9.x          | 1.9.x          | Enabled  | Do not upgrade Snapshot Agent to the latest 1.9.x. | See [Upgrade Path for Consul Clients and Servers 1.9.x - ACLs Enabled](faq#upgrade-path-for-consul-clients-and-servers-1-9-x-acls-enabled).   |
+| 1.9.x          | 1.9.x          | Disabled | Snapshot Agent affected by upgrade.                | See [Upgrade Path for Consul Clients and Servers 1.9.x - ACLs Disabled](faq#upgrade-path-for-consul-clients-and-servers-1-9-x-acls-disabled). |
+| 1.8.x          | 1.8.x          | Enabled  | Do not upgrade Snapshot Agent to the latest 1.8.x  | See [Upgrade Path for Consul Clients and Servers 1.8.x - ACLs Disabled](faq#upgrade-path-for-consul-clients-and-servers-1-8-x-acls-enabled).  |
+| 1.8.x          | 1.8.x          | Disabled | Snapshot Agent affected by upgrade.                | See [Upgrade Path for Consul Clients and Servers 1.8.x - ACLs Disabled](faq#upgrade-path-for-consul-clients-and-servers-1-8-x-acls-disabled). |
+
```

Comment:
I would pull out the snapshot agent column out of this table. Its mentioned explicitly below for each section where its clear that when ACLs are disabled, we do require upgrading both the snapshot agent and clients/servers to the latest in the 1.9.x series.